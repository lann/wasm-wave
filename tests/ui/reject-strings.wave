// Reject surrogates.
string("\u{d800}")
string("\u{dbff}")
string("\u{dc00}")
string("\u{dcff}")
string("\u{d800}\\u{dc00}")
// Reject invalid values.
string("\u{110000}")
string("\u{ffffffff}")
string("\u{80000000}")
// Reject invalid syntax.
string("\u{-1}")
string("\u{+1}")