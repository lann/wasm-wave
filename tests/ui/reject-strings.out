// Reject surrogates.
invalid character escape at 7..17
invalid character escape at 7..17
invalid character escape at 7..17
invalid character escape at 7..17
invalid character escape at 7..26
// Reject invalid values.
invalid character escape at 7..19
invalid token at 7..16
invalid token at 7..16
// Reject invalid syntax.
invalid token at 7..8
invalid token at 7..8
